addons/website/static/src/scss/website.scss

```diff
@@ -426,6 +426,45 @@ table.table_desc tr td {
 // Snippets
 //
 
+// To Check data-vcss
+.s_title {
```

Comment:
@sbuhl I have said like 5 times at both you and fabien: are you **sure** you want to remove this as it is working fine? ... and here we reintroduce it?

cc @sdegueldre :smile:

Reply:
😳 I'm sorry 
"faire, défaire et refaire..."

Reply:
IMO this should not be done with classes but with theme customization. Titles are central to a theme's design and they should be consistent across the board. If you want to give the user control, give them control at the theme level (like we do for font and color)

Reply:
Nothing to do with theme design to me. It's not a style for all titles, just a one-time title style you need for a specific page. Anyways, we'll check again when this branch remaining commits are moved at the right location (cc @Cocographique :+1:)